nuget.config

```diff
@@ -5,11 +5,12 @@
   </config>
   <packageSources>
     <clear />
-    <add key="GVFSBuildDependencies" value="https://www.myget.org/F/gvfsbuilddependencies/api/v3/index.json" />
+    <add key="VFSForGitBuildDependencies" value="https://vfsforgit.myget.org/F/build-dependencies/api/v3/index.json" />

```

Comment:
I'm not seeing this feed in with my user.  Have we pushed any packages to this new feed?  Do we need to update the builds to push to it?

Reply:
Yes, we'll need to update the relevant builds to push to the new feed.

Reply:
@kewillford do you see the feed when you log into vfsforgit.myget.org?

Reply:
I've mirrored the permissions from the old feed. Hopefully that fixes the issue.